app/styles/templates/project/thankyou.scss

```diff
@@ -1,14 +1,27 @@
 .thankyou-container {
-  .icon, .header, .text, .project-link{
+  .icon, .header, .text, .project-link {
     width: 100%;
     text-align: center;
   }
 
-  .icon img {
-    display: inline-block;
+  .header, .text {
+    margin-bottom: 20px;
+  }
+
+  .header {
+    font-size: 1.5em;
```

Comment:
I like that you're using ems and I think that's probably the way going forward when the app becomes responsive.

With our baseline at 14px 1.5em is rendered as 21px, the Sketch reference has headers set at 24px (1.7em approx.).

Minor conversation at this point but hey, every pixel counts. 😛